chart/templates/mayastor/call-home/call-home-deployment.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if .Values.call-home.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-call-home
+  labels:
+    app: call-home
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: call-home
+  template:
+    metadata:
+      labels:
+        app: call-home
+        openebs.io/logging: "true"
```

Comment:
Do we want supportability tooling to capture this log by default?

Reply:
more than this, by default the Logging infra will capture & store it's logs by default for the set retention period

Reply:
No, this component does not need logging. Good catch, thanks.